src/wrapped/wrappedlibc_private.h

```diff
@@ -307,7 +307,7 @@ GOW(erand48_r, iFppp)
 GOM(err, vFEipV)
 //DATAB(errno, 
 GO(__errno_location, pFv)
-GOW(error, vFiipV)
+//GOWM(error, vFiipV)
```

Comment:
This one should not be disabled, and a wrapped "my_" should be created, prformation the fmt using vsprintf in a local buffer kind of call